ansible/idr-mineotaur.yml

```diff
@@ -11,6 +11,15 @@
       owner: 1000
       state: directory
 
+  - name: set default rsync mineotaur_url
+    set_fact:
+      default_rsync_mineotaur_url: >-
+        rsync://{{
+          hostvars[groups[idr_environment |
+            default('idr') + '-omero-hosts'][0]]
```

Comment:
To ensure this playbook can be run on it's own you'll need to ensure you've loaded the hostvars for `...-omero-hosts` by adding a preceding (empty) playbook

Reply:
Similar to https://github.com/IDR/deployment/blob/master/ansible/idr-kubernetes.yml#L6 you mean ?

Reply:
Exactly.

Reply:
See b8e1254